src/Components/ComponentsNoDeps.slnf

```diff
@@ -0,0 +1,35 @@
+{
+  "solution": {
+    "path": "Components.sln",
+    "projects": [
```

Comment:
What is this json madness. Give me back my unintelligible format full of GUIDs

Reply:
I know. Who will think of our job security? With a file format like this, *anyone* could just come along and understand it 😱